lib/tabula.ex

```diff
@@ -61,6 +62,11 @@ defmodule Tabula do
     def keys(row), do: row |> Keyword.keys
   end
 
+  defimpl Row, for: Any do
+    def get(row, col, default \\ nil), do: row |> Map.get(col, default)
```

Comment:
IMO we should be more strict here, to make it crash sensibly:

```elixir
def get(%{__struct__: _}, col, default \\ nil), do: ...
```

Since we don't really handle `Any` per se, but need some way to distinguish between a map and a struct. WYT?